scipy/integrate/_ivp/radau.py

```diff
@@ -304,12 +317,14 @@ def __init__(self, fun, t0, y0, t_bound, max_step=np.inf,
 
         self.jac_factor = None
         self.jac, self.J = self._validate_jac(jac, jac_sparsity)
+        self.nlusove = 0
```

Comment:
I have added this counter for the number of forward-backward substitutions. During my development I used is to ensure the computational effort remained acceptable. I guess it could probably be used as an additional attribute to the `OdeResult` class in the future.

Reply:
```suggestion
        self._nlusove = 0
```
Consider the preceding underscore to indicate that it's supposed to be private. (Even if that's not already done in the existing code, it's good practice.)

Reply:
I have performed that change. In the future, it would be interesting to make BDF and Radau return that value as part of the OdeSolution result, as it is a good indicator of the computational cost.